File: Data Structures & Algorithms/two-integer-sum/submission-6.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash = {}
        for i in range(len(nums)):
            missing = target - nums[i]
            if missing in hash:
                return [hash[missing], i]
            else:
                hash[nums[i]] = i

File: Data Structures & Algorithms/duplicate-integer/submission-6.cpp
#include <set>
#include <vector>
#include <iostream>
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        set<int> set1(nums.begin(), nums.end());

        if ((nums).size() != (set1).size()){
            return true;
        } else{
            return false;
            }

    }
};